WISE-Community/WISE-Client
# chore(VLE): Convert component.ts to Angular

## Changes
Removed unused attributes like mode and instead hard-code it to 'student' in the component itself since only students use this component.

## Test
- Launch preview and student vle
- Make sure all data loads: prompt, saved work (if any), imported work, teacher annotations
- Importing work as background (e.g. draw -> concept map)

Closes #443
